src/spatch/backend_system.py

```diff
@@ -560,10 +568,10 @@ def visit(node, order, _visiting={}):
             del _visiting[node]
             order[node] = None  # add sorted node
 
-        to_sort = list(graph.keys())
+        # to_sort = list(graph.keys())  # @eriknw to @seberg: should this be used?
         order = {}  # dict as a sorted set
         for n in list(graph.keys()):
```

Comment:
```suggestion
        order = {}  # dict as a sorted set
        for n in graph:
```
Not just was that a "maybe I will mutate" probably, but the for loop also looks just like "not cleaned up" to me...

EDIT: (First thought just to remove the new comment.)